uni/lib/view/course_unit_info/course_unit_info.dart

```diff
@@ -23,6 +23,14 @@ class CourseUnitDetailPageView extends StatefulWidget {
 
 class CourseUnitDetailPageViewState
     extends SecondaryPageViewState<CourseUnitDetailPageView> {
+  @override
+  void initState() {
```

Comment:
Can't you achieve this simply calling the construction? This field should be private, I think.